recipes/cibots/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+    name: cibots
+    version: 0.1
+
+source:
+    url: https://github.com/UV-CDAT/ci-bots/archive/0.1.tar.gz
+    sha256: c16d96619efa86fdabc23f49b6bff04a78e1d99a097a5fa8320504038dd6976b
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - requests
+
+test:
+  command:
+    - python run_tests.py -v2
+
+about:
+    home:  http://github.com/UV-CDAT/ci-bots
+    license: BSD
```

Comment:
Needs a version as well. Is it 2-Clause or 3-Clause?

Also the upstream repo doesn't seem to even note this is BSD in `setup.py`. 😟 Might want to take a look at that. Would also add a license file to the source repo and add it to the `MANIFEST.in` as well.